src/components/foodDescription/FoodDescription.js

```diff
@@ -0,0 +1,37 @@
+// Libraries
+import PropTypes from 'prop-types'
+import { useMemo } from 'react'
+
+// lodash
+import _keys from 'lodash/keys'
+
+// components
+import Sidebar from "./components/sidebar";
+import FoodList from './components/foodList';
+
+// css
+import './foodDescription.css'
+
+
+function FoodDescription(props) {
+    
+    const { foodList } = props;
+    const keys = useMemo(() => _keys(foodList),[foodList])
+
+    return (
+        <div className="flex food-description">
+            <Sidebar categories={keys} />
+            <FoodList foodList={foodList} />
+        </div>
+    )
+}
+
+FoodDescription.propTypes = {
+    foodList : PropTypes.object
```

Comment:
write the complete shape too

Reply:
i havent given it because different retaurant can have different property names. 
some can have like {starters:[], breakfast:[]}, and others as {recommended:[], platters:[]}